aten/src/ATen/TensorUtils.cpp

```diff
@@ -151,6 +151,18 @@ void checkAllSameGPU(CheckedFrom c, ArrayRef<TensorArg> tensors) {
   checkAllSame(c, tensors, checkSameGPU);
 }
 
+void checkTensorsSameDevice(CheckedFrom c, const TensorArg& t1, const TensorArg& t2) {
```

Comment:
just wondering if why this is named  `checkTensorsSameDevice` instead of `checkSameDevice` (seem to have better correspondence with `checkSameGPU` )  . But both names are fine with me :)

Reply:
Thank you for taking a look! I agree that `checkSameDevice` would be a better name. There's a `checkSameDevice` in `LinearAlgebraUtils.h`, so I had thought another `checkSameDevice` might cause some confusion.

I'll rename it asap.